mods/seasonal/moves.js

```diff
@@ -109,6 +109,52 @@ let BattleMovedex = {
 		type: "Electric",
 		zMovePower: 200,
 	},
+	// Andy
+	pilfer: {
+		accuracy: 100,
+		basePower: 70,
+		category: "Physical",
+		desc: "",
+		shortDesc: "",
+		id: "pilfer",
+		name: "Pilfer",
+		pp: 5,
+		priority: 1,
+		flags: {protect: 1, mirror: 1, contact: 1},
+		onPrepareHit: function (target, source) {
+			this.attrLastMove('[still]');
```

Comment:
No anim, needs to be sucker punch/night slash.

Reply:
Happens on OnTry to prevent the animation from failing if the move fails

Reply:
Isn't that a good thing though? Same thing happens with sucker punch